pkg/cmd/codespace/codespace_selector.go

```diff
@@ -25,6 +26,7 @@ func AddCodespaceSelector(cmd *cobra.Command, api apiClient) *CodespaceSelector
 
 	cmd.PersistentFlags().StringVarP(&cs.codespaceName, "codespace", "c", "", "Name of the codespace")
 	cmd.PersistentFlags().StringVarP(&cs.repoName, "repo", "R", "", "Filter codespace selection by repository name (user/repo)")
+	cmd.Flags().StringVar(&cs.repoOwner, "repo-owner", "", "To filter by the `username` of the repository owner")
```

Comment:
Just a suggestion to make the descriptions more consistent with existing flags:

```suggestion
	cmd.Flags().StringVar(&cs.repoOwner, "repo-owner", "", "Filter codespace selection by repository owner (username or org)")
```

Reply:
Sounds better, thanks.